tests/e2e/tour/tour.spec.js

```diff
@@ -9,10 +11,93 @@ const expectWithinMargin = async (actual, expected, margin) => {
   })
 }
 
+test.describe('When clicking the "Skip for now" button', () => {
+  test.beforeEach(async ({ page, context }) => {
+    await setupTests(page, context, false)
+
+    await page.route(/collections.json/, async (route) => {
+      await route.fulfill({
+        json: singleCollection.body,
+        headers: singleCollection.headers
+      })
+    })
+
+    await page.goto('/search')
+  })
+
+  test('should close the tour', async ({ page }) => {
+    // Expect the first step to show the "Take the tour" button
+    await expect(page.locator('.search-tour__welcome')).toContainText('Welcome to Earthdata Search!')
+
+    // Click the "Skip for now" button to close the tour
+    await page.click('button:has-text("Skip for now")')
```

Comment:
Clicking Skip for now has to happen in a beforeEach block. This test should just verify that it is closed.

The reason it has to happen in a beforeEach is because the test for refreshing the page has to have clicked the Skip button before it gets to the test. That does not happen if it is clicked within a test block, but only if it is skipped in a beforeEach block